docs/UserGuide.adoc

```diff
@@ -421,7 +426,9 @@ There are three categories by which you can sort the matches, namely:
 +
 
 
-These categories are further elaborated below. Note that **only one category can be used in one sort command**.
+These categories will be further explained below.
```

Comment:
If I'm not wrong, the phrase is 'sort by criteria' and not 'sort by category'. Can @TheRealRavager confirm if I'm correct?

Reply:
changed